speculos/mcu/display.py

```diff
@@ -61,7 +62,7 @@ def can_read(self, screen: DisplayNotifier) -> None:
     'SYLVE_CYAN': 0x29f3f3,
 }
 
-
+from time import sleep
```

Comment:
put higher with the other imports

Reply:
It was used only for my tests, I removed it.